cmd/amp/commands.go

```diff
@@ -128,7 +129,7 @@ func addCommands(cmd *cobra.Command, c cli.Interface) {
 
 func info(c cli.Interface) error {
 	s := c.Server()
-	c.Console().Infof("[%s]\n", s)
+	fmt.Fprintf(c.Err(), "[%s]\n", s)
```

Comment:
Sorry, my brain's slow today, but still not clear to me ... is this for printing informational stuff or errors? If for errors, the func is poorly named. If for information, we shouldn't be printing that to stderr. Maybe I'm missing something here? (I get that we only want to print IDs to stdout when it quiet mode...)